index.html

```diff
@@ -0,0 +1,169 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Proud 2 Pay</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <!-- Top nav -->
+        <div class="top-nav">
+            <ul class="nav-menu">
+                <li class="nav-items">
+                    <a href="#" class="nav-link">
+                        <img src="imgs/facebook-f-48.svg" alt="facebook">
+                    </a>
+                </li>
+                <li class="nav-items">
+                    <a href="#" class="nav-link">
+                        <img src="imgs/twitter-48.svg" alt="twitter">
+                    </a>
+                </li>
+                <li class="nav-items">
+                    <a href="#" class="nav-link">English</a>
+                </li>
+                <li class="nav-items">
+                    <a href="#" class="nav-link">My Page</a>
+                </li>
+                <li class="nav-items">
+                    <a href="#" class="nav-link">Logout</a>
+                </li>
+            </ul>
+        </div>
+
+        <!-- Main Nav -->
+        <div class="mobileNavBar">
+            <span class="closeIcon">x</span>
+            <ul class="mobileNavBar2">
+                <li><a href="#home" class="nav-link">Home</a></li>
+                <li><a href="#About" class="nav-link">About</a></li>
+                <li><a href="#Program" class="nav-link">Program</a></li>
+                <li><a href="#Sponsor" class="nav-link">Sponsor</a></li>
+                <li><a href="#News" class="nav-link">News</a></li>
+                <li><a href="#Campaign" class="nav-link">Campaign</a></li>
+            </ul>
+        </div>
+
+        <nav class="navbar">
+            <a href="#home" class="nav-brand">Digital Adept
+                <span>TM</span>
+            </a>
+            <ul class="nav-menu-main">
+                <li class="nav-item">
+                    <a href="#home" class="nav-link first-child">Home</a>
+                </li>
+                <li class="nav-item">
+                    <a href="about.html" class="nav-link">About</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#program" class="nav-link">Program</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#sponsor" class="nav-link">Sponsor</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#news" class="nav-link">News</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#campaign" class="nav-link">CB Campaign</a>
+                </li>
+            </ul>
+            <!-- Hamburger -->
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
```

Comment:
- Kindly make sure that the hamburger menu works as expected when we click on it disappeared and we option with transparent background and with a closing button that allows it to be closed. please fix the design of the list of options and also the functionality :+1: